test/sequential/test-module-loading.js

```diff
@@ -35,59 +35,70 @@ assert.strictEqual(process.mainModule, module,
 // assert that it's *not* the main module in the required module.
 require('../fixtures/not-main-module.js');
 
-// require a file with a request that includes the extension
-const a_js = require('../fixtures/a.js');
-assert.strictEqual(42, a_js.number);
+{
+  // require a file with a request that includes the extension
+  const a_js = require('../fixtures/a.js');
+  assert.strictEqual(a_js.number, 42);
+}
 
-// require a file without any extensions
-const foo_no_ext = require('../fixtures/foo');
-assert.strictEqual('ok', foo_no_ext.foo);
+{
+  // require a file without any extensions
+  const foo_no_ext = require('../fixtures/foo');
+  assert.strictEqual(foo_no_ext.foo, 'ok');
+}
 
 const a = require('../fixtures/a');
-const c = require('../fixtures/b/c');
 const d = require('../fixtures/b/d');
 const d2 = require('../fixtures/b/d');
-// Absolute
-const d3 = require(path.join(__dirname, '../fixtures/b/d'));
-// Relative
-const d4 = require('../fixtures/b/d');
 
-assert.strictEqual(false, false, 'testing the test program.');
+{
+  const c = require('../fixtures/b/c');
+  // Absolute
+  const d3 = require(path.join(__dirname, '../fixtures/b/d'));
+  // Relative
+  const d4 = require('../fixtures/b/d');
 
-assert.ok(a.A instanceof Function);
-assert.strictEqual('A', a.A());
+  assert.strictEqual(false, false, 'testing the test program.');
```

Comment:
That should probably be in `test-assert.js` and inside an `assert.doesNotThrow()`. Especially since this is a first-time contribution, I'm OK leaving it. Maybe we can open an issue for it and label it `good first contribution` for someone else to fix. :-D